community/GOVERNANCE.md

```diff
@@ -1,18 +1,35 @@
 # Argo Governance
+This document outlines the governance for the overall Argo Project and the four sub-projects contained within.  
 
-Describes the roles and responsibilities and governing structure of the Argo Community.
+## Roles and Membership
+Roles are described in the [Community Membership](https://github.com/argoproj/argoproj/blob/master/community/membership.md) document
+For the purpose of governance, reviewers, approvers and leads are considered maintainers.
+The list of current maintainers, their project and organizational affiliation(s)is maintained in the [MAINTAINERS](https://github.com/argoproj/argoproj/MAINTAINERS.md) file.
 
-## Technical Oversight Committee
 
-The Argo Technical Oversight Committee (TOC) is responsible for cross-cutting product and design decisions. The membership of the TOC consists of leads of the Argo projects and technical representatives from the steering committee.
+## Conflict Resolution and Voting
+The vast majority of issues are resolved by consensus. If consensus cannot be reached, issues are resolved by voting. All decisions should be recorded publicly in GitHub or meeting notes and recordings.
 
-## Steering Committee
+* A vote requires a simple majority to pass. 
+* Voting is done by adding +1/-1 on the PR submitted for the issue. 
+* Each maintainer gets a binding vote. Non-binding votes from the community are encouraged. 
```

Comment:
This item is left out of the "Changes to governance" section.

Reply:
Fixed